tools/serverpod_cli/lib/src/analyzer/models/validation/restrictions.dart

```diff
@@ -770,6 +775,25 @@ class Restrictions {
       ];
     }
 
+    var field = definition.findField(parentNodeName);
+
+    if (field == null) return [];
+
+    if (field.type.isListType) {
+      var referenceClass = modelRelations
+          ?.findAllByClassName(field.type.generics.first.className)
+          .firstOrNull;
+
+      if (referenceClass?.moduleAlias != definition.moduleAlias) {
+        return [
+          SourceSpanSeverityException(
+            'A List relation is not allowed on external tables.',
```

Comment:
Question: Do we use "external" consistently as terminology for depency tables? Isn't "module" more in line with the users understanding at when doing this configuration?

Reply:
Yes true this is probably more correct, I think my idea was that it could be modules or Serverpod tables. But I guess you could argue that Serverpod is a module as well.